.gitignore

```diff
@@ -10,10 +10,11 @@ package.json
 utils/get_links.py
 
 #Data 
-**/data/raster_data/*
+sentinel1/data/raster_data/*
+itslive/data/raster_data/single_glacier_itslive.zarr/*
 
 #Extra nbs
-sentinel1/subste_nbs
+sentinel1/subste_n
```

Comment:
these are no longer present, they were from figuring out the subset / full time series approach for s1 tutorial, i think it can be removed ?

Reply:
^also removing the last two characters from that line was a mistake

Reply:
i took this line out